r/configure

```diff
@@ -49,7 +49,8 @@ if [ "$INCLUDE_DIR" ] || [ "$LIB_DIR" ]; then
 elif [ "$PKGCONFIG_CFLAGS" ] || [ "$PKGCONFIG_LIBS" ]; then
   echo "Found pkg-config cflags and libs!"
   PKG_CFLAGS=${PKGCONFIG_CFLAGS}
-  PKG_LIBS=${PKGCONFIG_LIBS}
+  #PKG_LIBS=${PKGCONFIG_LIBS}
+  PKG_LIBS="-larrow -lparquet" # hard coded! what is the right way to do this?
```

Comment:
I think this should rather be addressed by updating the `PKG_CONFIG_NAME` higher

Reply:
Having this seems to work for me: 

```
PKG_CONFIG_NAME="arrow parquet"
```

This needs an update to the `README.Rmd` (and `README.md`) to have `-DARROW_PARQUET=ON`, i.e.: 

```
cmake .. -DARROW_PARQUET=ON -DCMAKE_BUILD_TYPE=Release -DARROW_BOOST_USE_SHARED:BOOL=Off
```